gke/grpc-bookstore.yaml

```diff
@@ -16,20 +16,50 @@
 # and the container for the Extensible Service Proxy (ESP) to
 # Google Kubernetes Engine (GKE).
 
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: esp-grpc-bookstore
+  annotations:
+    kubernetes.io/ingress.class: "gce"
+    kubernetes.io/ingress.allow-http: "false"
+spec:
+  tls:
+  - hosts:
+    - SERVICE_NAME
+    secretName: esp-ssl
+  backend:
+    serviceName: esp-grpc-bookstore
+    servicePort: 80
```

Comment:
Since it is using SSL, should we change it to port 443? I know it doesn't matter, but we should follow conventions.